Server/ConcordiaCurriculumManager/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,125 @@
+﻿using AutoMapper;
+using ConcordiaCurriculumManager.DTO;
+using ConcordiaCurriculumManager.Models;
+using ConcordiaCurriculumManager.Services;
+using ConcordiaCurriculumManager.Swagger;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using Swashbuckle.AspNetCore.Filters;
+using System.ComponentModel.DataAnnotations;
+
+namespace ConcordiaCurriculumManager.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthenticationController : Controller
+{
+    private readonly IMapper _mapper;
+    private readonly ILogger<AuthenticationController> _logger;
+    private readonly IUserAuthenticationService _userAuthorizationService;
+
+    public AuthenticationController(IMapper mapper, ILogger<AuthenticationController> logger, IUserAuthenticationService authorizationService)
+    {
+        _mapper = mapper;
+        _logger = logger;
+        _userAuthorizationService = authorizationService;
```

Comment:
I think this should be `_userAuthenticationService` and `authenticationService`.